src/main/java/org/jboss/threads/Version.java

```diff
@@ -48,11 +50,20 @@ private Version() {
         }
         JAR_NAME = jarName;
         VERSION_STRING = versionString;
-        try {
+        boolean logVersion = AccessController.doPrivileged((PrivilegedAction<Boolean>) Version::shouldLogVersion).booleanValue();
```

Comment:
The way you did it, if you have an exception here, you won't log anything. Which is not the preexisting behavior.

But I will let you judge of if it's a problem or not.

Reply:
TBH I really want the default to be `false`; anyway I prefer to err on the side of the more desirable behavior.